atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -494,6 +506,7 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
+                    'config': config,
```

Comment:
If we don't have a v2 schema 2 config object we use use to fill this entry in, it should be omitted altogether (as currently).

In other words it should be present and populated, or not present at all.